### DISCUSSION TITLE
Can-D in Discord

### DISCUSSION THREAD

**Discussion by DanM3rcurius** [SecretDecks/Documentation#118] [gov.channel]
### Regarding SourceCred and Can-D generation in Discord
![image](https://user-images.githubusercontent.com/80257662/135984991-b9a2ff18-218f-434f-aef7-fecc8b7823c7.png)

Right now the default weight for any emoji given is 1.
The Can-D Pill (the lowest of the custom emojis) has a weight of 2.

### Up for discussion/ Question
1) Should we keep it like this or remove the default weight and only have Can-D generated with the custom emojis?
2) Will the -1 break stuff or work properly? 
3) Should we change the -1 to 0.5 so that it halves the amount of Can-D?

**Comment by wlstylr** [comment]
I think we should keep the default emoji weight.  I think leaving thumbs ups, smiley faces, etc helps with the general moral of the discord community and makes people feel seen.  

Unsure about the -1, but I do recall decimals breaking SourceCred in the past.
